modules/detectron/batch_permutation_op.h

```diff
@@ -21,6 +21,7 @@
 #include "caffe2/core/logging.h"
 #include "caffe2/core/operator.h"
 #include "caffe2/utils/math.h"
+#include <cstring>
```

Comment:
Put the include files of in front of others. Please refer to https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes